<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>615</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="videoBtn">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>180</y>
     <width>341</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>휴먼편지체</family>
     <pointsize>30</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(220, 252, 248);</string>
   </property>
   <property name="text">
    <string>영상촬영   </string>
   </property>
  </widget>
  <widget class="QPushButton" name="skipBtn">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>320</y>
     <width>341</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>휴먼편지체</family>
     <pointsize>30</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 235, 252);</string>
   </property>
   <property name="text">
    <string>넘어가기   </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>200</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/next_button.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>340</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/next_button.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>videoBtn</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>changePage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>661</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>735</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>skipBtn</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>changePage2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>666</x>
     <y>377</y>
    </hint>
    <hint type="destinationlabel">
     <x>740</x>
     <y>365</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changePage()</slot>
  <slot>changePage2()</slot>
 </slots>
</ui>
